[Misc]pass the active batch size through logger

frank-wei opened the pull request:
Summary: The FINAL_ONLY in sampling parameter will only output completed requests. When we want to query the active requests/batch size, we can not directly get it through len(outputs). A better way is to pass it through the logger.

Differential Revision: D86217922

zhuohan123 reviewed (changes requested):
Should we log this field directly? Instead of having it as a public class field for others to access.

frank-wei committed:
pass the active batch size through logger (#28105)

Summary:

The FINAL_ONLY in sampling parameter will only output completed requests. When we want to query the active requests/batch size, we can not directly get it through len(outputs). A better way is to pass it through the logger.

Differential Revision: D86217922

Signed-off-by: Wei Wei <wwei6@meta.com>

frank-wei committed:
pre commit

Signed-off-by: Wei Wei <wwei6@meta.com>
